tasks.py

```diff
@@ -73,13 +73,20 @@ def install_minimum(c):
     with open('pyproject.toml', 'rb') as pyproject_file:
         pyproject_data = tomli.load(pyproject_file)
 
-    dependencies = pyproject_data.get('project', {}).get('dependencies', [])
+    project_data = pyproject_data.get('project', {})
+    dependencies = project_data.get('dependencies', [])
+    optional_dependencies = project_data.get('optional-dependencies', {})
+    pomegranate_dependencies = optional_dependencies.get('pomegranate', [])
+
     python_version = '.'.join(map(str, sys.version_info[:2]))
     minimum_versions = _get_minimum_versions(dependencies, python_version)
+    minimum_pomegranate_versions = _get_minimum_versions(pomegranate_dependencies, python_version)
 
     if minimum_versions:
         install_deps = ' '.join(minimum_versions)
+        pomegranate_deps = ' '.join(minimum_pomegranate_versions)
         c.run(f'python -m pip install {install_deps}')
+        c.run(f'python -m pip install --force-reinstall --no-cache-dir {pomegranate_deps}')
```

Comment:
perhaps the pomegranate logic should only be run if `minimum_pomegranate_versions` isn't empty